attestation-service/attestation-service/src/policy_engine/mod.rs

```diff
@@ -13,6 +15,40 @@ pub enum PolicyEngineType {
     OPA,
 }
 
+#[derive(Error, Debug)]
+pub enum OpaError {
```

Comment:
I think this shold be in the opa module, no?

Reply:
Renamed it to RegoError and keeping it here, so that we dont have to include opa module things here.

Reply:
I would still move it into the `opa` module, rego == opa, the module name is not 100% correct anymore since we replaced OPA with regorus. the errors are specific to the `opa` module. e.g. if we move opa/rego code behind a feature toggle at a later point, we don't have to untangle the rego errors from the generic `policy_engine` mod, also.